atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -94,13 +95,26 @@ public BackgroundSweeperImpl(LockAwareTransactionManager txManager,
                                  Supplier<Long> sweepPauseMillis,
                                  Supplier<Integer> sweepBatchSize,
                                  SweepTableFactory tableFactory) {
+        this(txManager, kvs, sweepRunner, isSweepEnabled, sweepPauseMillis, sweepBatchSize, tableFactory,
+                Optional.absent());
+    }
+
+    public BackgroundSweeperImpl(LockAwareTransactionManager txManager,
```

Comment:
That would be an API breaking change. Internal products use the existing constructor (although we do intend to add in a custom perf logger when this is available).  I can do this as long as it is considered okay to break the API here.